packages/react-components/react-teaching-popover/stories/src/TeachingPopover/TeachingPopoverCarouselText.stories.tsx

```diff
@@ -23,22 +23,22 @@ export const CarouselText = () => (
     </TeachingPopoverTrigger>
     <TeachingPopoverSurface>
       <TeachingPopoverHeader>Tips"</TeachingPopoverHeader>
-      <TeachingPopoverCarousel defaultValue="test-1">
-        <TeachingPopoverCarouselCard value="test-1">
+      <TeachingPopoverCarousel defaultValue="1">
+        <TeachingPopoverCarouselCard value="1">
```

Comment:
Nit: If this is for narration, would something like "Carousel Card 1" be appropriate or are plain numbers preferred??

Reply:
I did `{index => <TeachingPopoverCarouselNavButton aria-label={`Tip ${index}`} />}`, for most of them, so they should read as "Tip 1", mostly

This one just has the static text "X of Y" and no buttons, so I didn't actually need to change these. I just figured it didn't hurt and was consistent